andela/firestar-backend
# 167730743 Ft social login authentication via facebook and google

#### What does this PR do?
* Implement login feature via Google and Facebook
#### Description of Task to be completed?
* create a controller.
* create endpoint route for social authentication.
#### How should this be manually tested?
* git clone `https://github.com/andela/firestar-backend.git`
* cd `firestar-backend`
* Checkout to the branch `git checkout ft-login-social-auth-167730743`
* run `npm install`
* Setup test databases, see `.env.example` for sample 
         - run `npm run createdb:dev` for development
        -  run `npm run createdb:test` for test
**P.S** `JWT_SECRET_KEY=pass a string of characters e.g THISISBAREFOOTNOMADPROJECT`
* Run migration to create tables 
         - run `npm run migrate:dev` for development
        -  run `npm run migrate:test` for test
* Populate test database: ` npm run seed:test`
* To run the test  `npm run test`

* Start the sever by runnning `npm run start:dev`
#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
* N/A
#### Screenshots or gifs (if appropriate)
* N/A
#### Questions:
* N/A
